src/Manifolds.jl

```diff
@@ -949,4 +949,14 @@ export get_basis,
 # atlases and charts
 export get_point, get_point!, get_parameters, get_parameters!
 
+# additional functions on Simplex
```

Comment:
Is it reasonable to export these functions or are they more of an internal part?

Reply:
(outside of Manifolds.jl)